ocr/packages/ocr/acceptImage/build.sh

```diff
@@ -1,6 +0,0 @@
-#!/bin/bash
-set -e
-rm -f index.js .include
-npm install
-npm run build
-echo 'index.js' > .include
```

Comment:
Don’t need a new .include?

Reply:
There already is a committed `.include` with the right contents and that contents is invariant.   So, the step to write it in the build is unnecessary.